ports/stm32/system_stm32.c

```diff
@@ -538,7 +540,7 @@ void SystemClock_Config(void)
   }
 #endif
 
-    uint32_t vco_out = RCC_OscInitStruct.PLL.PLLN * (HSE_VALUE / 1000000) / RCC_OscInitStruct.PLL.PLLM;
+    uint32_t vco_out = RCC_OscInitStruct.PLL.PLLN * (HSI_VALUE / 1000000) / RCC_OscInitStruct.PLL.PLLM;
```

Comment:
This is going to need a config or something because `HSE_VALUE` should be used if HSE is used.